test/test_browser.py

```diff
@@ -1992,7 +1992,8 @@ def test_emscripten_main_loop_and_blocker_exit(self):
   @parameterized({
     '': ([],),
     'worker': (['--proxy-to-worker'],),
-    'pthreads': (['-pthread', '-sPROXY_TO_PTHREAD'],)
+    'pthreads': (['-pthread', '-sPROXY_TO_PTHREAD'],),
+    'strict': (['-sSTRICT'],)
```

Comment:
Can you add  a trailing comma here to make future additions just a one line change?